internal/testhelpers/httprecorder/recorder.go

```diff
@@ -37,7 +37,14 @@ func CreateRecordedScwClient(cassetteName string) (*scw.Client, *recorder.Record
 		recorderMode = recorder.ModeRecording
 		config, err := scw.LoadConfig()
 		if err != nil {
-			return nil, nil, err
+			// The nightly tests should run without config by using the credentials in the environment
+			if os.Getenv("RUNNING_NIGHTLY_TESTS") == "true" {
+				config = &scw.Config{
+					Profile: *scw.LoadEnvProfile(),
```

Comment:
Could you have a merge with LoadConfig and LoadEnvProfile ? and not depend on the environment variable